sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQuerySinkMetrics.java

```diff
@@ -40,10 +41,10 @@
  *
  * <p>In general metrics be in the namespace 'BigQuerySink' and have their name formatted as:
  *
- * <p>'{baseName}-{metricLabelKey1}:{metricLabelVal1};...{metricLabelKeyN}:{metricLabelValN};'
+ * <p>'{baseName}*{metricLabelKey1}:{metricLabelVal1};...{metricLabelKeyN}:{metricLabelValN};'
  */
 public class BigQuerySinkMetrics {
-  private static Boolean supportMetricsDeletion = false;
+  private static boolean supportMetricsDeletion = false;
```

Comment:
if this is a constant should be styled as 
`private static final boolean SUPPORT_METRICS_DELETION = false;`

Reply:
This value is mutable